pgcli/pgexecute.py

```diff
@@ -177,7 +177,8 @@ def run(self, statement):
             # Check if the command is a \c or 'use'. This is a special
             # exception that cannot be offloaded to `pgspecial` lib. Because we
             # have to change the database connection that we're connected to.
-            if sql.startswith('\c') or sql.lower().startswith('use'):
+            command = sql.split()[0]
+            if command == '\c' or command.lower() == 'use':
```

Comment:
Can you add a check for `command == '\connect'` ? 

That was the reason I had it set to startswith `\c` but you're right I shouldn't generalize too much.